advanced/am-pattern-3/templates/am/gateway/wso2am-pattern-3-am-gateway-deployment.yaml

```diff
@@ -128,3 +131,7 @@ spec:
       - name: wso2am-pattern-3-am-gateway-conf
         configMap:
           name: wso2am-pattern-3-am-gateway-conf
+      - name: wso2am-pattern-3-conf-entrypoint
```

Comment:
@Chathuru why do we need to override the entry point here?

Reply:
fix https://github.com/wso2/kubernetes-apim/pull/365/commits/3588e9f5044c3947741ef019ecda0ee150daad18